tutorials/13_diagnosis_sbc.ipynb

```diff
@@ -0,0 +1,1897 @@
+      "text/plain": [
+       "<Figure size 576x360 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "posterior_ = BiasedPosterior(posterior, shift = -.1)\n",
+    "ranks, dap_samples = run_sbc(thetas, xs, posterior_, num_workers=1)\n",
+    "check_stats = check_sbc(ranks, thetas, dap_samples, 1_000)\n",
+    "print(check_stats)\n",
+    "f, ax = sbc_rank_plot(ranks, 1_000, plot_type = \"hist\", num_bins=30)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "A similar behavior is observed, but this time we see an upshot of ranks to higher values. Because we have shifted the expected value of the posterior to smaller values, we see an upshot in high ranks.\n",
+    "It is interesting to see that the historgams obtained provide very convincing evidence that this is not a uniform distribution. "
```

Comment:
It great to show both cases. I think the conclusion can be stronger, or say more directly that this shows that the shape of SBC rank dists corresponds to certain biases in the posterior: left-skewed -> systematic underestimation of the posterior mean, right-skewed -> over-estimation etc. I think this should be a key insight for the reader here.